src/mplhep/styles.py

```diff
@@ -1,5 +1,34 @@
+import sys
+from matplotlib.pyplot import style as plt_style
+
 # Short cut to all styles
 from .styles_cms import CMS, CMSTex, ROOT, ROOTTex, firamath, fabiola
 from .styles_atlas import ATLAS
 from .styles_alice import ALICE
 from .styles_lhcb import LHCb, LHCbTex
+
+
+def set_style(styles):
+    """
+    Set the experiment specific plotting style
+
+    Example:
+
+        >>> import mplhep as hep
+        >>> hep.set_style("ATLAS")
+        >>> hep.set_style(mplhep.style.CMS)
+
+    Parameters
+    ----------
+        styles (`str` or `mplhep.style` `dict`): The experiment style
+    """

```

Comment:
I took a look at how you do docstrings, but if I messed up the style please let me know so I can fix it!